client/src/main/scala/drt/client/components/FlightComponents.scala

```diff
@@ -51,8 +51,9 @@ object FlightComponents {
 
     val paxNos = List(
       <.p(s"Pax: $portDirectPax (${flight.apiFlight.ActPax.getOrElse(0)} - ${flight.apiFlight.TranPax.getOrElse(0)} transfer)"),
-      <.p(s"Max: $max")
-    ) :+ flight.totalPaxFromApiExcludingTransfer.map(p => <.span(s"API: $p")).getOrElse(EmptyVdom)
+      <.p(s"Max: $max"),
+      <.p(s"List totalPax : ${flight.apiFlight.TotalPax}")
```

Comment:
We'll want to remove this `List`

Reply:
I will be removing before merging . Just kept it for quick look for any issue in preprod test.